client/src/app/core/services/data-send.service.ts

```diff
@@ -18,22 +18,14 @@ export class DataSendService {
      *
      * @param http The HTTP Client
      */
-    public constructor(private http: HttpClient) {}
+    public constructor(private http: HttpService) {}
```

Comment:
variable name `http` is usually blocked for HttpClient. Rename to `hs` or `httpService`
The comment is also wrong.